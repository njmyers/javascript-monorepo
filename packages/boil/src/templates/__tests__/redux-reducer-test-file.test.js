import reduxReducerTestFile from "../redux-reducer-test-file";

describe("it works", () => {
  test("it is a function ", () => {
    expect(typeof reduxReducerTestFile).toBe("function");
  });

  test("it runs without crashing", () => {
    expect(reduxReducerTestFile("test-component")).toMatchSnapshot();
  });
});
