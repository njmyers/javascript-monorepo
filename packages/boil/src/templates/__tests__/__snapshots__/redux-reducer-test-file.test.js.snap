// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works it runs without crashing 1`] = `
"/** @flow */
import testComponentReducer from '../test-component-reducer';
import * as testComponentActions from '../test-component-actions';

describe('it should dispatch actions', () => {
  test('it is a function type', () => {
    expect(typeof testComponentReducer).toBe('function')
  });
})
"
`;
